### PR TITLE
Add sha256 & string to doxygen & cppcheck meta.yml

### PR DESCRIPTION
**Pull request by FarnazH**
A few minor updates:
1. Add `sha256` to meta.yml of cppcheck and doxygen. 
2. Add  `string` option to meta.yml of cppcheck and doxygen, updated packages on <https://anaconda.org/theochem/repo> so that the tar filenames look nicer.
3. Add a short summary for doxygen.

### CODE REVIEW

File: tools/cppcheck-conda-recipe/meta.yaml [hunk 1]
```diff
@@ -3,10 +3,12 @@ package:
   version: '1.80'
 source:
   fn: 1.80.tar.gz
+  sha256: 20863db018d69c33648bdedcdc9d81d818b9064cc4333f0d4dc45e114bd0f000
   url: https://github.com/danmar/cppcheck/archive/1.80.tar.gz
 
 build:
   number: 3
+  string: 1
```
Comment (tovrstra) [inline tools/cppcheck-conda-recipe/meta.yaml]: I guess the purpose is to keep the `string` and `number` identical, 4 in this case. The cppcheck recipe has undergone already some changes, including yours, it becomes 4.

File: tools/doxygen-conda-recipe/meta.yaml [hunk 2]
```diff
@@ -6,11 +6,13 @@ package:
 source:
   url: ftp://ftp.stack.nl/pub/users/dimitri/doxygen-1.8.13.src.tar.gz
   fn: doxygen-1.8.13.src.tar.gz
+  sha256: af667887bd7a87dc0dbf9ac8d86c96b552dfb8ca9c790ed1cbffaa6131573f6b
   patches:
   - find_iconv.patch
 
 build:
   number: 1
+  string: 1
```
Comment (tovrstra) [inline tools/doxygen-conda-recipe/meta.yaml]: idem (2)

**Comment by FarnazH** [comment]
@tovrstra thanks for your comments. I changed the numbers based on your comments, please let me know if there are any mistakes. I updated the files on <https://anaconda.org/theochem/repo>, but unfortunately, I have deleted the earlier builds (with smaller `number` and `string` values). I hope that is fine, as there were no major changes in the recipes. Please let me know.

**Comment by tovrstra** [comment]
Looks good. Getting rid of the old files is fine. The old version are not very useful to keep in this case. It would only be different for historical reasons or to guarantee reproducibility of some results. That's certainly not the case here. Merging...

**Pull request merged**